processor/attributes/testdata/config.yaml

```diff
@@ -0,0 +1,117 @@
+processors:
+  # The following example demonstrates the default behavior of insert a key/value.
+  # There is no need to specify action because the default action is insert.
+  attributes/default:
+    actions:
+      # The following inserting a new attribute {"attribute1": 123} to spans where
+      # the key "attribute1" doesn't exist.
+      # This demonstrates how to backfill spans with an attribute that may
+      # not have been sent by all clients.
+      - key: "attribute1"
+        value: 123
```

Comment:
I think it is worth to add somewhere a mention about the type of the value being inferred from the configuration, as the old add-attribute processor.